### PR TITLE
docs: updates the documentation for the ToH tutorial more comprehensi…

### LINKED ISSUE #52961: tutorials
### Describe the problem that you experienced

The command `ng new` in angular 17 needs to have '--no-standalone' for the `ng new` to create the app.module.ts and app-routing.module.ts files.

In this tutorial https://angular.io/tutorial/tour-of-heroes/toh-pt0, when we have to run the command `ng new angular-tour-of-heroes`, it doesn't say that we have to specify '--no-standalone'

### Enter the URL of the topic with the problem

https://angular.io/tutorial/tour-of-heroes/toh-pt0

### Describe what you were looking for in the documentation

just add '--no-standalone' for users that are in angular 17

### Describe the actions that led you to experience the problem

doing the tutorial

### Describe what you want to experience that would fix the problem

_No response_

### Add a screenshot if that helps illustrate the problem

_No response_

### If this problem caused an exception or error, please paste it here

_No response_

### If the problem is browser-specific, please specify the device, OS, browser, and version

_No response_

### Provide any additional information here in as much as detail as you can

_No response_

### PR DESCRIPTION
**Pull request by aparzi**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

Issue Number: fixes #52961

## What is the new behavior?
 Update documentation

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by aparzi** [comment]
Hi @AndrewKushnir any news for this PR?

**Comment by AndrewKushnir** [comment]
@MarkTechson just wanted to see if you have some thoughts on how to communicate the fact that `ng new` generates standalone version since v17 and the ToH guides rely on NgModule-based structure.

**Comment by AndrewKushnir** [comment]
> Hi @AndrewKushnir any news for this PR?

@aparzi thanks for creating this PR. I've added Mark as a reviewer to see if we need to communicate this more in Tour of Heroes tutorial. Thank you.

**Comment by aparzi** [comment]
> > Hi @AndrewKushnir any news for this PR?
> 
> @aparzi thanks for creating this PR. I've added Mark as a reviewer to see if we need to communicate this more in Tour of Heroes tutorial. Thank you.

@AndrewKushnir ok thanks. Available for any modification or update.

**Review by MarkTechson — CHANGES_REQUESTED** [review]
Thanks for this feedback, left a suggestion. If you can add this then I think we'll be good to go.

File: aio/content/tutorial/tour-of-heroes/toh-pt0.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Create a new project
 
-Use the `ng new` command to start creating your **Tour of Heroes** application.
+Use the `ng new` command to start creating your **Tour of Heroes** application. <br>
+<b>Please note:</b> From version 17 of Angular a standalone project will be created by default. To create a project with the old scaffolding you need to use the option `ng new --no-standalone`
```
[the remark this answers is not in the document]
Comment (aparzi) [reply aio/content/tutorial/tour-of-heroes/toh-pt0.md]: Hi @MarkTechson, thanks for your review. I applied the requested changes

**Review by MarkTechson — APPROVED** [review]
LGTM, thanks so much for this contribution!

**Comment by aparzi** [comment]
> LGTM, thanks so much for this contribution!

it's a pleasure

**Review by AndrewKushnir — APPROVED** [review]

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit e3dbadd9a496c076f32da965d709d17314ad7dd7.

**Comment by Tweniee** [comment]
I can quickly create a new pull request (PR) that incorporates the current changes and the requested changes. This would streamline the process and minimize any delays. Would you like me to proceed with creating the new PR?

**Comment by aparzi** [comment]
> I can quickly create a new pull request (PR) that incorporates the current changes and the requested changes. This would streamline the process and minimize any delays. Would you like me to proceed with creating the new PR?

I have already done so and the PR with my changes and modification requests has already been accepted

**Pull request closed without merging**